tests/test_actions.py

```diff
@@ -779,11 +785,8 @@ def test_passthru_attribute(self) -> None:
     def test_passthru_attribute_missing(self) -> None:
         a = FakeAction()
         q = Silently(a)
-        msg = "SilentlyPerformable(FakeAction) has no attribute 'desribe'"
-        with pytest.raises(AttributeError) as exc:
-            q.desribe()
-
-        assert str(exc.value) == msg
+        with pytest.raises(AttributeError):
+            q.not_a_real_attribute_of_silent_performables()
```

Comment:
I understand the test to a call for a method that doesn't exist.. but I'm not sure I understand why the old test isn't also included?  It was meant to test the fact that `FakeAction` doesn't have a describe for `Silently` to utilize.